lib/monitoring/aws-elasticache/ElastiCacheClusterMonitoring.ts

```diff
@@ -46,6 +48,11 @@ export interface ElastiCacheClusterMonitoringOptions
    */
   readonly addCpuUsageAlarm?: Record<string, UsageThreshold>;
```

Comment:
We should clarify that this still useful in Redis contexts.

Reply:
done (see below)